nautobot/extras/api/views.py

```diff
@@ -599,11 +599,10 @@ def run(self, request, *args, **kwargs):
                 request.method, detail="This job's source code could not be located and cannot be run"
             )
 
-        job_queues = job_model.job_queues.all().values_list("name", flat=True)
+        job_queues = list(job_model.job_queues.all().values_list("name", flat=True))
         valid_queues = job_queues if job_queues else [settings.CELERY_TASK_DEFAULT_QUEUE]
```

Comment:
Could this just be `valid_queues = job_model.task_queues`?

Actually, is `valid_queues` even needed/used at all now?

Reply:
Yeah valid_queues is used after this. If a user specified a task queue,  valid_queues is used to determine whether the user-specified queue is valid or not.